Util/include/Poco/Util/ServerApplication.h

```diff
@@ -158,6 +172,11 @@ class Util_API ServerApplication: public Application
 		/// waitForTerminationRequest(), this method will return
 		/// and the application can shut down.
 
+	static void registerTerminateCallback(TerminateCallback tCB,
+		const std::string& message = _terminateMessage);
+		/// Registers a termination callback.
+		/// Used to notify all interested users about system shutdown.
```

Comment:
The description shall be rephrased IMO. The callback can perform any cleanup or whatever is required to do on shutdown, notifying users is only one of the possibilities.